# Solution Extra Exercise

M2 is strictly dominated by a mix putting $\alpha$ on U and $(1-\alpha)$ on D if the mix does strictly better against both L and R, that is if
$$\alpha e^{3r}+(1-\alpha) e^0>e^{2.6r}$$
and 
$$\alpha e^0+(1-\alpha)e^{3r}>e^{0.5r}.$$
To achieve this, note that the second inequality is in a sense easier to achieve. To find the critical $r$, we will therefore choose $\alpha$ such that it is satisfied with equality. That is, we take
$$\alpha=\frac{e^{0.5r}-e^{3r}}{1-e^{3r}}.$$
Plugging this \alpha into the first inequality, we solve for the critical $r$ by solving this inequality (with equality); that is we solve
$$\frac{e^{0.5r}-e^{3r}}{1-e^{3r}} e^{3r}+\left(1-\frac{e^{0.5r}-e^{3r}}{1-e^{3r}}\right) 1-e^{2.6r}=0.$$
I plot the left hand side as a function of $r$ below and calculate where it is equal to zero.

In [12]:
using Plots
using Roots

f(r)=(e^(0.5r)-e^(3r))/(1-e^(3r))*e^(3r)+(1-(e^(0.5r)-e^(3r))/(1-e^(3r)))-e^(2.6r)

println("Critical r is ", fzero(f,[0.01,0.1]))

plot(0.01:0.001:0.15,f)



Critical r is 0.08751853869308206


Hence, we conclude that for $r>0.08751853869308206$ M2 is strictly dominated by an appropriate mix (see $\alpha$ above) of U and R.